controller/pages/api/util/pewpew.ts

```diff
@@ -134,7 +136,17 @@ export async function postPewPew (parsedForm: ParsedForm, authPermissions: AuthP
       if (version === null) {
         return { json: { message: `${match[1]} is not a valid semver version: ${version}` }, status: 400 };
       }
-
+      // If latest version is being updated, write it to file:
+      if(latest){
+        const filename = "./.latestpewpewversionfile.ts";
+        const data = "export const latestPewPewInFile = {\"version\": \"" + version + "\"}; // " + new Date() + ";";
+        try {
+          fs.writeFileSync(filename, data);
```

Comment:
I thought we were going to read and write this to S3? Currently this will get wiped out if our controller goes unhealthy. The new controller will come up with the hard-coded file in the source tree.

We should have a cached variable that we can update then lazy write the updated file to S3. There's currently integration tests that call this code, the tests should be updated to validate that the file is written. See `PPaasS3File` in common to use to read and write.